pandas/core/apply.py

```diff
@@ -1798,7 +1798,7 @@ def normalize_keyword_aggregation(
 
 
 def _make_unique_kwarg_list(
-    seq: Sequence[tuple[Any, Any]]
+    seq: Sequence[tuple[Any, Any]],
```

Comment:
Yeah could you revert these style changes? And others in this PR

Reply:
Done, apologies, somehow it ran over files I never meant to touch.